src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,61 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = (
+  {
+    movie: {
+      title,
+      description,
+      imgUrl,
+      imdbUrl,
+    },
+  },
+) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl || 'https://m.media-amazon.com/images/'
+          + 'M/MV5BMjAxMzY3NjcxNF5BMl5BanBnXkFtZTcwNTI5OTM0Mw@@._V1_SX300.jpg'}
```

Comment:
You can create a variable for the default url to make your code cleaner

Reply:
fixed , Thank